### PR TITLE
Parser changes v2

### PR DESCRIPTION
**Pull request by KPP13**
Some fixes in this pull request. Spring added, so now it is possible to use the application as a REST service and also it is able to run cyclic task.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -7,7 +7,7 @@
     <groupId>g-shock-team.robot</groupId>
     <artifactId>robot-shock-robot</artifactId>
     <packaging>pom</packaging>
-    <version>1.0-SNAPSHOT</version>
+    <version>${project.version}</version>
```
Comment (maciejkocur) [inline pom.xml]: We were told that project version should be constant. Not in properties. Not sure why. But check if mvn during build is not throwing warning about that.

File: robot-db/pom.xml [hunk 2]
```diff
@@ -5,7 +5,7 @@
     <parent>
         <artifactId>robot-shock-robot</artifactId>
         <groupId>g-shock-team.robot</groupId>
-        <version>1.0-SNAPSHOT</version>
+        <version>${project.version}</version>
```
Comment (maciejkocur) [inline robot-db/pom.xml]: same as above

**Comment by maciejkocur** [comment]
Generally, well done. You have my approval. :+1:

File: pom.xml (hunk 1 above)
Comment (KPP13) [reply pom.xml]: Fixed - you can check this in the next commit. This should be constant value. If not, then maven generates a lot of warnings :)

File: robot-db/pom.xml (hunk 2 above)
Comment (KPP13) [reply robot-db/pom.xml]: fixed :)

**Comment by czyzu0507** [comment]
Accepted by me and Maciej

**Pull request merged**